multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/variables/Variables.java

```diff
@@ -583,6 +583,25 @@ public interface Variables {
                                                                                         .name("serviceBrokerNamesJobsIds")
                                                                                         .build();
 
+    Variable<Boolean> IS_SERVICE_BINDING_KEY_OPERATION_IN_PROGRESS = ImmutableSimpleVariable.<Boolean> builder()
+                                                                                            .name("isServiceBindingKeyOperationInProgress")
+                                                                                            .defaultValue(false)
+                                                                                            .build();
+
+    Variable<Boolean> IS_SERVICE_BINDING_KEY_OPERATION_ALREADY_DONE = ImmutableSimpleVariable.<Boolean> builder()
+                                                                                             .name("isServiceBindingKeyOperationAlreadyDone")
+                                                                                             .defaultValue(false)
+                                                                                             .build();
```

Comment:
I think renaming the variable to IS_SERVICE_BINDING_KEY_OPERATION_FINISHED (isServiceBindingKeyOperationFinished) might be better.